sklearn/metrics/_dist_metrics.pyx.tp

```diff
@@ -117,6 +118,8 @@ METRIC_MAPPING{{name_suffix}} = {
     'sokalmichener': SokalMichenerDistance{{name_suffix}},
     'sokalsneath': SokalSneathDistance{{name_suffix}},
     'haversine': HaversineDistance{{name_suffix}},
+    'arccos': ArccosDistance{{name_suffix}},
+    'angular': ArccosDistance{{name_suffix}},
```

Comment:
I think it is better to have only one specification for this distance

I find that `AngularDistance` is a better naming than `ArccosDistance` as it caries more meaning. Hence I would rename this `DistanceMetric` to `AngularDistance` and here have:

```suggestion
    'angular': AngularDistance{{name_suffix}},
```